enos/utils/enostask.py

```diff
@@ -81,6 +82,15 @@ def decorated(*args, **kwargs):
 
             # Constructs the environment
             kwargs['env'] = make_env(kwargs['--env'])
+
+            # Check if the environment has been loaded for the commands
+            # requiring it
+            if fn.__name__ not in ['up', 'deploy', 'info'] and \
```

Comment:
@msimonin If the idea is to produce contracts around functions, then why not leveraging the type system of Python3? 

In other case, if the idea is to ensure that the linked env is a file, I would recommend to move following lines after a `else` at line 40.